yarn-project/simulator/src/avm/journal/journal.ts

```diff
@@ -253,11 +253,12 @@ export class AvmPersistableStateManager {
     if (success) {
       this.acceptNestedCallState(nestedState);
     }
-    const functionName =
+    let functionName =
```

Comment:
You can revert this back to const after you undid certain changes

Reply:
Good catch!